xbmc/GUIPassword.cpp

```diff
@@ -445,13 +444,13 @@ bool CGUIPassword::CheckMenuLock(int iWindowID)
       break;
     case WINDOW_MUSIC_NAV:      // Music
       bCheckPW = profileManager->GetCurrentProfile().musicLocked();
-      if (!bCheckPW && strMediasourcePath != "") // check mediasource by path
-        return g_passwordManager.IsMediaPathUnlocked(strMediasourcePath, "music");
+      if (!bCheckPW && !GetMediasourcePath().empty()) // check mediasource by path
```

Comment:
"Mediasource" is not an english word, proper camel casing looks like "MediaSource"

Reply:
hm, thats what i'm thinkin bout since jan. "Mediasource" looks dumb